Task50/Task50/AlertsTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Task50
+{
+    [TestFixture]
+    public class AlertsTest
+    {
+        private static readonly string URL = "https://www.seleniumeasy.com/test/javascript-alert-box-demo.html";
+        private IWebDriver _driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            _driver = new ChromeDriver { Url = URL };
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+        }
+
+        [TearDown]
+        public void TearDownTest()
+        {
+            _driver.Quit();
+        }
+
+        [Test]
+        public void JavaScriptConfirmBoxClickedYesTest()
+        {
+            _driver.FindElement(By.XPath("//button[@onclick='myConfirmFunction()']")).Click();
+            var alert = _driver.SwitchTo().Alert();
+            alert.Accept();
+
+            var actualValue = _driver.FindElement(By.Id("confirm-demo")).Text;
+            var expectedValue = "You pressed OK!";
+
+            Assert.AreEqual(actualValue, expectedValue);
```

Comment:
Add the third parameter to see additional information in case if test fail on this step.